src/parameterizations/lateral/MOM_hor_visc.F90

```diff
@@ -468,81 +472,92 @@ subroutine horizontal_viscosity(u, v, h, diffu, diffv, MEKE, VarMix, G, GV, US,
 
     ! Get barotropic velocities and their gradients
     call barotropic_get_tav(BT, ubtav, vbtav, G, US)
+
     call pass_vector(ubtav, vbtav, G%Domain)
 
-    do j=js-1,je+2 ; do i=is-1,ie+2
+    ! Calculate the barotropic horizontal tension
+    do j=Jsq-2,Jeq+2 ; do i=Isq-2,Ieq+2
```

Comment:
This is tracer point loop, so I suspect that the right loop ranges should be `do j=js-2,je+2 ; do i=is-2,ie+2`.